tools/prow-config-updater/config/config.go

```diff
@@ -110,7 +110,14 @@ func UpdateProw(env ProwEnv, dryrun bool) error {
 	return helpers.Run(
 		fmt.Sprintf("Updating Prow configs with command %q", updateCommand),
 		func() error {
-			out, err := cmd.RunCommand(updateCommand)
+			kf := os.Getenv("GOOGLE_APPLICATION_CREDENTIALS")
+			if kf != "" {
+				authCommand := "gcloud auth activate-service-account --key-file=" + kf
+				if _, err := cmd.RunCommand(authCommand); err != nil {
+					return fmt.Errorf("error activating service account with %q", kf)
```

Comment:
don't we want to print error?

Reply:
We don't need to, if there is an error, it will be shown as a comment on the pull request.